crawl-ref/source/dat/des/arrival/large.des

```diff
@@ -332,8 +332,8 @@ FTILE:  .t1 = floor_moss
 KMASK:  Ww = no_monster_gen
 MAP
 ttttttttttttttttttttttttttttttttttttttttttt
-tttttttttttt1ttttttwwtttttttttttttttttttttt
-tttt...1ttttt..tttwwwttt.tttttttt....ttttt
+ttttttttt1.t1ttttttwwtttttttttttttttttttttt
+tttt...1..t....tttwwwttt.tttttttt....ttttt
```

Comment:
This is a good change, I think this vault wold also benefit from a second exit path past the river.